### PR TITLE
add new functionality

### PR DESCRIPTION
**Pull request by ViachaslauKabak**
* Add labels
* Add webhooks
* Create group
*  Create project
*  Create master user

### CODE REVIEW

**Review by vutkin — CHANGES_REQUESTED** [review]
Please resolve conflicts.

File: tasks/install.yml [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+    - not gitlab_file.stat.exists
+    - molecule_test
+
+# Start and configure GitLab. Sometimes the first run fails, but after that,
+# restarts fix problems, so ignore failures on this run.
+- name: "Reconfigure GitLab (first run)."
+  command: >
+    gitlab-ctl reconfigure
+    creates=/var/opt/gitlab/bootstrapped
+  failed_when: False
```
Comment (vutkin) [inline tasks/install.yml:81]: Should we add retry here trying to install during first run?

- shell: /usr/bin/foo
  register: result
  until: result.stdout.find("all systems go") != -1
  retries: 5
  delay: 10

File: tasks/install.yml [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+  become: True
+
+- name: "Create default user if was not created automatically"
+  command: >
+    gitlab-rails runner -e production
+    /opt/gitlab/embedded/service/gitlab-rails/db/fixtures/production/001_admin.rb
+  args:
+    removes: /usr/bin/gitlab-rails
+  become: True
+  failed_when: False
```
Comment (vutkin) [inline tasks/install.yml:91]: should we replace to block - rescue?

File: tasks/main.yml [hunk 3]
```diff
@@ -1,129 +1,32 @@
...
+    - gitlab_version != "latest"
 
-- name: "Task on CI issue"
-  shell: nohup /opt/gitlab/embedded/bin/runsvdir-start </dev/null >/dev/null 2>&1 &
-  args:
-    executable: "/bin/bash"
-    removes: /usr/bin/gitlab-ctl
+- name: "Set api to v4"
+  set_fact:
+    gitlab_api_version: "v4"
```
Comment (vutkin) [inline tasks/main.yml:14]: We still need it? It already set to `v4` in defaults.

File: tasks/install.yml [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+  args:
+    removes: /usr/bin/gitlab-ctl
+  become: True
+  when: (gitlab_mysql_enabled) and (gitlab_edition == "ee")
+  changed_when: False
+
+# SElinux for gitlab
+- name: "Detecting selinux"
+  include: selinux-gitlab.yml
+  when: ansible_selinux.status == "enabled" and ansible_selinux.mode != "disabled"
```
Comment (vutkin) [inline tasks/install.yml]: Should we user example from snippets for selinux?

**Review by vutkin — CHANGES_REQUESTED** [review]

File: tasks/install.yml (hunk 1 above)
Comment (ViachaslauKabak) [reply tasks/install.yml:81]: No - it fails sometimes - so the second run will be the same - failed

File: tasks/install.yml (hunk 2 above)
Comment (ViachaslauKabak) [reply tasks/install.yml:91]: it fails when used is in the list
so it works when user created

File: tasks/main.yml (hunk 3 above)
Comment (ViachaslauKabak) [reply tasks/main.yml:14]: it is double check - if user set v3 manually

File: tasks/install.yml (hunk 4 above)
Comment (ViachaslauKabak) [reply tasks/install.yml]: fixed

**Review by vutkin — APPROVED** [review]

**Pull request merged**